src/node_dotenv.cc

```diff
@@ -1,4 +1,6 @@
 #include "node_dotenv.h"
+#include <regex>  // NOLINT(build/c++11)
```

Comment:
Why NOLINT?

Reply:
Without NOLINT I encounter this error during `make-cpp`: 
`<regex> is an unapproved C++11 header.  [build/c++11] `
Same as [here](https://github.com/nodejs/node/blob/main/deps/v8/test/cctest/test-disasm-regex-helper.h#L10)